pkg/agent/controller/agent.go

```diff
@@ -497,8 +499,11 @@ func (a *Controller) filterLocalEndpointSlices(obj runtime.Object, op syncer.Ope
 	labels := endpointSlice.GetObjectMeta().GetLabels()
 
 	if labels[discovery.LabelManagedBy] != lhconstants.LabelValueManagedBy {
+		klog.Infof("Retuning nil since not managed by lighthouse %v", obj)
 		return nil, false
 	}
 
+	klog.Infof("In filterLocalEndpointSlices returning %v", obj)
```

Comment:
This will generate too many log entries in full run, esp in headless and stateful where we have multiple endpoints.